### PR TITLE
add unicode support for pretty.

### PR DESCRIPTION
**Pull request by arisliang**

### CODE REVIEW

File: omegaconf/container.py [hunk 1]
```diff
@@ -296,7 +296,7 @@ def pretty(self, resolve=False):
         :return: A string containing the yaml representation.
         """
         container = OmegaConf.to_container(self, resolve=resolve, enum_to_str=True)
-        return yaml.dump(container, default_flow_style=False)
+        return yaml.dump(container, default_flow_style=False, allow_unicode=allow_unicode)
```
Comment (omry) [inline omegaconf/container.py]: I think it's safe to just allow unicode.
no need to extend the interface for this.

**Comment by omry** [comment]
Also, please add a news snippet file to news.
Something like:

*news/110.bugfix*
Fixed pretty to handle strings with unicode characters correctly

**Comment by omry** [comment]
Oh, and of course - a unit test verifying this remains fixed.
(It should reproduce the problem you are fixing here, and should fail if your fix is not in place).

Thanks!

File: omegaconf/container.py (hunk 1 above)
Comment (arisliang) [reply omegaconf/container.py]: Removed the interface extension.

**Comment by arisliang** [comment]
`news/110.bugfix` - added news
unit test:

**Comment by omry** [comment]
Can you try to rebase on master? there still seems to be merge errors.

**Comment by arisliang** [comment]
There seems to too many conflicts when rebasing. I try re-create the PR

**Pull request closed without merging**